main.py

```diff
@@ -48,8 +48,7 @@ def main():
     else:
         graph = original_graph
 
-    # If a starting node was given as an argument, start there. Else pass None and one will be randomly selected.
-    start = int(args.start) if args.start else None
+    start = args.start if args.start else None
```

Comment:
Now I'm wondering if this is necessary at all? Is `args.start == None` if it is not set when executing? I think we can leave it too though, looks nice and explicit.

Reply:
Oh dang. I'm not too familiar with Python 😬. I think you're correct. I'll test and update in a bit.